packages/js/product-editor/src/blocks/product-fields/regular-price/regular-price-menu-group.tsx

```diff
@@ -0,0 +1,161 @@
+/**
+ * External dependencies
+ */
+import { createElement, Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { recordEvent } from '@woocommerce/tracks';
+import { MenuItem } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { TRACKS_SOURCE } from '../../../constants';
+import { handlePrompt } from '../../../utils';
+import { SetListPriceMenuItem } from '../../../components/variations-table/set-list-price-menu-item';
+import { VariationQuickUpdateMenuGroup } from '../../../components/variations-table/variation-actions-menus/variation-quick-update-menu-group';
+
+function isPercentage( value: string ) {
+	return value.endsWith( '%' );
+}
+
+function parsePercentage( value: string ) {
+	const stringNumber = value.substring( 0, value.length - 1 );
+	if ( Number.isNaN( Number( stringNumber ) ) ) {
+		return undefined;
+	}
+	return Number( stringNumber );
+}
+
+export function addFixedOrPercentage(
+	value: string,
+	fixedOrPercentage: string,
+	increaseOrDecrease: 1 | -1 = 1
+) {
+	if ( isPercentage( fixedOrPercentage ) ) {
+		if ( Number.isNaN( Number( value ) ) ) {
+			return 0;
+		}
+		const percentage = parsePercentage( fixedOrPercentage );
+		if ( percentage === undefined ) {
+			return Number( value );
+		}
+		return (
+			Number( value ) +
+			Number( value ) * ( percentage / 100 ) * increaseOrDecrease
+		);
+	}
+	if ( Number.isNaN( Number( value ) ) ) {
+		if ( Number.isNaN( Number( fixedOrPercentage ) ) ) {
+			return undefined;
+		}
+		return Number( fixedOrPercentage );
+	}
+	return Number( value ) + Number( fixedOrPercentage ) * increaseOrDecrease;
+}
+
+export const RegularPriceMenuGroup = () => (
+	<VariationQuickUpdateMenuGroup
+		name="list-price"
+		label={ __( 'List price', 'woocommerce' ) }
+	>
+		{ ( { selection, onChange, onClose } ) => {
+			const ids = selection.map( ( { id } ) => id );
+			return (
+				<>
+					<SetListPriceMenuItem
+						selection={ selection }
+						onChange={ onChange }
+						onClose={ onClose }
+					/>
+					<MenuItem
```

Comment:
Shouldn't these be `VariationQuickUpdateMenuItem` components? To guarantee consistency in how menu items are rendered/handled and allow extensions to place menu items in-between them?

Let's say that we decide in the future to render quick update menu items with a component other than `MenuItem`... if these were using `VariationQuickUpdateMenuItem` then we wouldn't need to do anything.

Reply:
Currently what is called `VariationQuickUpdateMenuItem` is a component that returns Fills. You're proposing that we should create another component that acts as a wrapper for what is currently the `MenuItem`, is that correct? Because we couldn't use `VariationQuickUpdateMenuItem` as it is, in this scenario here.